.github/workflows/release.yaml

```diff
@@ -158,13 +158,14 @@ jobs:
       - name: download-and-upload-tarball
         env:
           GITHUB_TOKEN: ${{ secrets.GIT_UPLOAD_TOKEN }}
+          GOPATH: ${HOME}/go
```

Comment:
Is this where `GO` is installed in a github action environment?

Reply:
No, I think Go is not installed in a github action environment but `install_yq.sh` creates the `GO` directory at the following line. Hence, we can use it.
https://github.com/kata-containers/kata-containers/blob/main/ci/install_yq.sh#L24

Reply:
This will work, but this is error-prone (**not** your fault @ManaSugi).

I'm pretty sure we'll change that script at some point and the change will byte us quite hard. :-/

In any case, this is good enough for now.